### ISSUE TITLE
Border: Input Fields e Counter

### ISSUE THREAD

**Issue by AnaMontes11** [Telefonica/mistica-design#2062]
## Description
Currently, the input border uses the "border" token (Grey 3 #DDDDDD), which is not accessible, so the goal is to darken this element.

There are two possible approaches:

1. Change inputs to use the "control" token (Grey 4 #949494).
2. Create a specific token for these cases.

We need to list the elements affected by this change.
Additionally, we should review dark mode.

Discussion and POC: Border: Input Fields & Counter #2058

* We need material to communicate the differenciations to obs

## Tasks

- [x] Update specs
   - [x] Update input specs
   - [x] [Update counter specs](https://www.figma.com/design/LhZ5g2KAckJLzBIA1720H0/branch/6OxuFbekTf80lY3y8bnJFy/%F0%9F%94%B8-Counter-Component-Specs?m=auto&node-id=0-1&t=CClPbDufEYrwpHsl-1)
- [ ] Design documentation
- [x] Add to mobile library
- [x] Add to desktop library
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-15952)
- [x] [iOS ticket](https://jira.tid.es/browse/IOS-11208)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2230)
